runtime/caml/minor_gc.h

```diff
@@ -54,9 +54,8 @@ struct caml_minor_tables {
 
 struct domain;
 
-CAMLextern void caml_minor_collection (void);
+CAMLextern asize_t global_minor_heap_wsz_per_domain;
 
-#ifdef CAML_INTERNALS
```

Comment:
Is the removal of this `CAML_INTERNALS` guard intentional? 
Aren't some of these functions meant to be guarded to not be visible elsewhere.